src/onboarding/integration-badge.ts

```diff
@@ -19,6 +19,7 @@ class IntegrationBadge extends LitElement {
     return html`
       <div class="icon">
         <img
+          alt=""
```

Comment:
The title is rendered just below, therefore I believe this image to be decorative.